boardswarm-cli/src/ui.rs

```diff
@@ -206,14 +269,23 @@ pub async fn run_ui(
                         Input::Up => { terminal.scroll_up(); },
                         Input::Down => { terminal.scroll_down(); },
                         Input::ScrollReset => { terminal.scroll_reset(); },
-                        _ => (),
+                        Input::ShowMode => { terminal.show_mode(); },
+                        _ => { terminal.hide_mode(); },
+                    }
+                }
+                Some(new_state) = state_rx.recv() => {
+                    if new_state == "on" {
+                        terminal.set_power_on("PowerOn".to_string());
```

Comment:
I think it makes sense to decouple the mode from definition of how mode is displayed. By that I mean that if e.g. "off" is displayed as black on red, maybe other kinds of modes will also want to be displayed black on red, so I called this kind of display "NonFunctional", as opposed to "Functional" which is black on green. That is why the modes are not relayed 1:1 to the ui. On top of that, the string needs to be parsed to know how to display it. Not great maybe. And then if localization (i.e. displaying in national languages rather than English) is added, then the strings passed will not be known up front to the programmer, so the programmer will effectively be unable to parse (in a sane way) even if they wanted to.

For now this question boils down to "where should the mode string be parsed in order to decide how the mode is displayed?". Any preference/rationale for that?